auth-api/email_templates/govm_member_invitation_email.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
```

Comment:
Do we need a separate template for this? Can't we use the one existing?

Reply:
the IDIR username and password wording is different...